pkg/gui/custom_commands.go

```diff
@@ -151,10 +153,65 @@ func (gui *Gui) handleCustomCommandKeybinding(customCommand config.CustomCommand
 						return gui.surfaceError(err)
 					}
 
+					return gui.createMenu(title, menuItems, createMenuOptions{showCancel: true})
+				}
+			case "menuFromCommand":
+				f = func() error {
+                                        // Collect cmd to run from config
+			                cmdStr, err := gui.resolveTemplate(prompt.Command, promptResponses)
+			                if err != nil {
+			                	return gui.surfaceError(err)
+			                }
+
+                                        // Collect Filter regexp
+			                filter, err := gui.resolveTemplate(prompt.Filter, promptResponses)
+			                if err != nil {
+			                	return gui.surfaceError(err)
+			                }
+
+                                        // Run and save output
+                                        message,err := gui.GitCommand.RunCommandWithOutput(cmdStr)
+			                if err != nil {
+			                	return gui.surfaceError(err)
+			                }
+
+					// Need to make a menu out of what the cmd has displayed
+                                        var candidates []string
+                                        reg := regexp.MustCompile(filter)
+                                        for _,str := range strings.Split(string(message), "\n"){
+                                            cand := ""
+                                            if str != "" {
```

Comment:
Can you reverse this check to:
```go
if str == "" {
  continue
}
```

Reply:
Can I ask if this a matter of style in Golang or are there any technical consequences?

Reply:
Just to make the code more readable.
It has no consequences.